tests/test_MocLogin.py

```diff
@@ -0,0 +1,101 @@
+from django.test import TestCase
+from core.models import AtmosphereUser as User
+from core.models import Provider
+from core.models import PlatformType
+from core.models import ProviderType
+
+from threepio import logger
+
+from atmosphere import settings
+
+
+# This is assuming a clean database
+class MocLogin(TestCase):
+    username = test_settings['username']
```

Comment:
xuhang56 - I really don't understand what you are saying.  I am still trying to figure out how what you are saying relates to testing our use case.